charts/datadog/templates/_container-system-probe.yaml

```diff
@@ -151,4 +154,10 @@
 {{- if .Values.agents.volumeMounts }}
 {{ toYaml .Values.agents.volumeMounts | indent 4 }}
 {{- end }}
+  livenessProbe:
```

Comment:
Can we gate these behind a config value so that it can be disabled with one config?